pkg/apis/servicecatalog/types.go

```diff
@@ -357,9 +357,9 @@ type ServiceInstanceStatus struct {
 	// the service instance.
 	DashboardURL *string
 
-	// Checksum is the checksum of the ServiceInstanceSpec that was last successfully
-	// reconciled against the broker.
-	Checksum *string
+	// ReconciledGeneration is the generation of the instance that was last
+	// successfully reconciled.
```

Comment:
will it be clear to everyone that "reconciled" means "the controller is done doing its job regardless of whether it was success or not" ?  Or will some people think it means "only successfully processed"?  I kind of think we may want to make it clear for newbies:
```
// ReconciledGeneration is the 'Generation' of the instanceSpec that was last
// processed by the controller - regardless of whether the processing was
// success or not.
```

Reply:
I think it will be clear. If someone is in the codebase, they should understand what reconciliation is

Reply:
I like @duglin's comment here.  Status is for users, not just programmers.  Ultimately we'll be generating docs from these comments, so we should make sure things are clear to users.  I imagine this will also be more clear once we have a status field to show the current state that the broker knows about.

Reply:
I should have been more clear, I think "successfully reconciled" will be ambiguous.